README.md

```diff
@@ -1 +1,208 @@
+
+### Key Features <a name="key-features"></a>
+
+- Fetch Weather of all the cities
+- Filter weather data
+- Open city details and display extra weather informations
+- switch from Celcius to Fareneight display format
+- Set local city persistently in local storage
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://weather-app-xx49.onrender.com)
+
+Kindly find below the link to the video presenting the app and it's features.
+
+- [Video Explainning the Proect](https://www.loom.com/share/8c6d48aad1ca4c9fa61c904bf321803b)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Node.js installed on your local machine
+- npm or yarn package manager
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/chiztechnology/weather-app.git
+
+  cd weather-app
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  yarn install
+```
+
+or with npm:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  yarn start
+```
+
+or with npm:
+
+```sh
+  npm start
+```
+
+This should start the development server and open the application in your default web browser.
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  yarn test
+```
+
+or with npm:
+
+```sh
+  npm test
+```
+
+### Deployment
+
+You can deploy this project using:
+
+- Vercel
+- Netlify
+- Railway
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Israel CHIZUNGU**
+
+- GitHub: [chiztechnology](https://github.com/chiztechnology)
+- Twitter: [IsraelChizungu](https://twitter.com/IsraelChizungu)
+- LinkedIn: [IsraelChizungu](https://www.linkedin.com/in/israelchizungu/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Improve page styling
+- [ ] Persist in localstorage
+- [ ] Make the website responsive
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+ For Nelson Sakwa [Inspired by his original design and prototyping](https://www.behance.net/sakwadesignstudio)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221380355-ac621f55-420d-4f69-836c-d05ccf0a93c2.png)

**[Optional]**

- [ ] It would be nice if you add a message here for the viewers of your repo such as **If you like this project then don't forget to give a star :star: to this repo :+1:** to make your readme file descriptive and professional :blush: